tools/deploy/test/user-data.yml

```diff
@@ -0,0 +1,16 @@
+#cloud-config
+yum_repos:
+  osbuild:
+    name: osbuild
+    baseurl: "http://10.0.2.2:8000"
```

Comment:
Where does this IP address come from? Is it always the ip of the host machine available from the guest?

Reply:
Yeah, documented [here](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29). Just learned about this myself for this PR. I'll add a comment with a link in the file.

Reply:
Oops, Ondřej already merged it (thanks!)

I'll post a separate PR.